### ISSUE TITLE
Repeated words check flags consecutive images with the same alt text

### ISSUE THREAD

**Issue by tbroadley** [tbroadley/spellchecker-cli#83]
Hypothetical repro case:

```markdown
![word](https://example.com/a.png)
![word](https://example.com/b.png)
```

We probably wouldn't expect spellchecker-cli to fail on this because the alt texts of the two images aren't actually consecutive. But I believe it does fail on this.
